e2e/functionalities/peer-dependency-component.e2e.ts

```diff
@@ -0,0 +1,49 @@
+import path from 'path';
+import { expect } from 'chai';
+import fs from 'fs-extra';
+import Helper from '../../src/e2e-helper/e2e-helper';
+
+describe('set-peer', function () {
+  this.timeout(0);
+  let helper: Helper;
+  before(() => {
+    helper = new Helper();
+  });
+  after(() => {
+    helper.scopeHelper.destroy();
+  });
+  describe('a component is a peer dependency', () => {
+    let workspaceCapsulesRootDir: string;
+    before(() => {
+      helper.scopeHelper.reInitLocalScope();
+      helper.fixtures.populateComponents(2);
+      helper.command.setPeer('comp2', '0');
+      helper.command.install();
+      helper.command.build();
```

Comment:
wonder how build here is passing. shouldn't it try to install that comp2 and fail?